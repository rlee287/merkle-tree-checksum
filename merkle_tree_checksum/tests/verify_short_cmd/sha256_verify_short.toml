bin.name = "merkle_tree_checksum"
args = "verify-hash -- hash_out_short"
fs.sandbox = true