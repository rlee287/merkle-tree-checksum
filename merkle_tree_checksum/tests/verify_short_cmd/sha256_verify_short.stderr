================================= 16_byte_file =================================
Info: 16_byte_file hash matches
================================= 20_byte_file =================================
Info: 20_byte_file hash matches
================================== empty_file ==================================
Info: empty_file hash matches
